src/Uno.UWP/WinRTFeatureConfiguration.ArrayPool.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Uno
+{
+	public static partial class WinRTFeatureConfiguration
```

Comment:
It looks like the contents of the partial files got switched around - the array pool code should go in this file and the original file should remain unchanged.

Reply:
Woops :) Indeed!